fontforge/encoding.c

```diff
@@ -576,6 +576,69 @@ return( NULL );
 return( item );
 }
 
+/* Parse the GlyphOrderAndAliasDB file format */
+static Encoding *ParseGlyphOrderAndAliasDB(FILE *file) {
+    char* names[1024];
+    char buffer[256];
```

Comment:
I am not familiar with the file format. Is this limit of 255 characters in the specification?